<launch>
    <arg name="veh" doc="Name of vehicle. ex: megaman"/>
    <arg name="pkg_name" value="object_detection"/>
    <arg name="node_name" default="object_detection_node"/>
    <group ns="$(arg veh)">
        <node name="$(arg node_name)" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen">
            <remap from="object_detection_node/image" to="camera_node/image/compressed" />
            <remap from="object_detection_node/objects_detected" to="mqtt_bridge/obstacle"/>
            <remap from="object_detection_node/objects_detected_led_msg" to="led_signal_node/objects_detected_led_msg"/>
        </node>
    </group>
</launch>
